packages/core/app/components/utils/useSeedHistory.js

```diff
@@ -96,7 +96,7 @@ function undoReducer(state, action) {
 const getNewSeed = () => seedrandom(null, { pass: (_, seed) => ({ seed }) }).seed;
 
 export function useSeedHistory(functionName) {
-  const key = `--${functionName}-seed`;
+  const key = `mechanic_seed_${functionName}`;
```

Comment:
Given that the `mechanic_` prefix is used in multiple places now I think this should be a constant.

Reply:
✅